fiddler/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools<61",
+    "fiddler-client==1.0.6",
```

Comment:
Upgrade the version to the latest version 1.4.0

```suggestion
    "fiddler-client==1.4.0",
```

Reply:
Done